pulp_rpm/app/modelresource.py

```diff
@@ -490,7 +491,27 @@ class Meta:
         )
 
 
+class RpmRepositoryResource(RepositoryResource):
+    """
+    A resource for importing/exporting RPM repository entities.
+    """
+
+    def set_up_queryset(self):
+        """
+        Set up a queryset for RpmRepositories.
+
+        Returns:
+            A queryset containing one repository that will be exported.
+        """
+        return RpmRepository.objects.filter(pk=self.repo_version.repository)
```

Comment:
Where does `self.repo_version` come from?  Is it an alias for the latest repository version from the `RepositoryResource`?

Reply:
"What repo-version are we exporting?" is the basis of the import/export process. See https://github.com/pulp/pulpcore/blob/main/pulpcore/plugin/importexport.py#L16